Signal/src/environment/OWSPreferences.h

```diff
@@ -52,13 +52,15 @@ extern NSString *const PropertyListPreferencesKeyEnableDebugLog;
 - (BOOL)hasRegisteredVOIPPush;
 - (void)setHasRegisteredVOIPPush:(BOOL)enabled;
 
-
 - (BOOL)hasDeclinedNoContactsView;
 - (void)setHasDeclinedNoContactsView:(BOOL)value;
 
 - (void)setIOSUpgradeNagVersion:(NSString *)value;
 - (nullable NSString *)iOSUpgradeNagVersion;
 
+- (BOOL)areReadReceiptsEnabled;
+- (void)setAreReadReceiptsEnabled:(BOOL)value;
```

Comment:
TIOLI: are you opposed to using properties for these?

Reply:
Nice idea. I'll try to do it the next time I touch this file.